.github/workflows/releaseConfigurator.yaml

```diff
@@ -7,6 +7,11 @@ name: Configurator Release
 
 on:
   workflow_dispatch:
+    inputs:
```

Comment:
Could not this cause a different version is used for the changelog (the one computed from the unreleased entries)?

Reply:
Everything should be using `${{ steps.version.outputs.next-version }}` and we are overwriting such value since the RT command checks the environment variable. Nothing else should compute the version on its own